src/lib/header/Banner.svelte

```diff
@@ -0,0 +1,33 @@
+<small
+    ><p class="banner">
+        <slot />
+        {#if $$slots.link}<span class="spacer">|</span> <slot name="link" />{/if}
+    </p></small
+>
+
+<style lang="scss">
+    @use '../../../styles/vendor/rfs';
```

Comment:
Can't we add this to the `scss.prependData` in `svelte.config.js`?

Reply:
RFS isn't used nor needed in every component.

Reply:
with scss you don't need to do the `../` hell you can just do `styles/vendor/rfs`

Reply:
> RFS isn't used nor needed in every component.

Right but it gets removed from the build, that doesn't matter.

Reply:
```suggestion
    @use 'styles/vendor/rfs';
```

Reply:
Tried this before and it genuinely broke -- Tried it again after you commented this and it suddenly works. Not sure why it broke the first time. I assumed that the root directory would also be set in the include paths.

Reply:
It will resolve from whatever your `process.cwd()` is, so your cwd must've been wrong